src/main/java/io/openshift/launchpad/ui/osio/NewProjectWizard.java

```diff
@@ -0,0 +1,221 @@
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Parent;
+import org.jboss.forge.addon.maven.projects.MavenBuildSystem;
+import org.jboss.forge.addon.maven.resources.MavenModelResource;
+import org.jboss.forge.addon.projects.Project;
+import org.jboss.forge.addon.projects.ProjectFactory;
+import org.jboss.forge.addon.resource.DirectoryResource;
+import org.jboss.forge.addon.ui.context.UIBuilder;
+import org.jboss.forge.addon.ui.context.UIContext;
+import org.jboss.forge.addon.ui.context.UIExecutionContext;
+import org.jboss.forge.addon.ui.context.UINavigationContext;
+import org.jboss.forge.addon.ui.context.UIValidationContext;
+import org.jboss.forge.addon.ui.input.UISelectOne;
+import org.jboss.forge.addon.ui.metadata.UICommandMetadata;
+import org.jboss.forge.addon.ui.metadata.WithAttributes;
+import org.jboss.forge.addon.ui.result.NavigationResult;
+import org.jboss.forge.addon.ui.result.Result;
+import org.jboss.forge.addon.ui.result.Results;
+import org.jboss.forge.addon.ui.util.Categories;
+import org.jboss.forge.addon.ui.util.Metadata;
+import org.jboss.forge.addon.ui.wizard.UIWizard;
+
+import javax.inject.Inject;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Creates a new project
+ *
+ * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
+ */
+public class NewProjectWizard implements UIWizard
+{
+   List<Booster> boosters = new ArrayList<>();
+   @Inject
+   @WithAttributes(label = "Booster", required = true)
+   private UISelectOne<BoosterDTO> booster;
+   @Inject
+   private ProjectName named;
+
+   /*
+      @Inject
+      private Version version;
+
+   */
+   @Inject
+   private TopLevelPackage topLevelPackage;
+   @Inject
+   private BoosterCatalogServiceFactory catalogServiceFactory;
+   @Inject
+   private ProjectFactory projectFactory;
+   @Inject
+   private MavenBuildSystem mavenBuildSystem;
+
+   @Override
+   public void initializeUI(UIBuilder builder) throws Exception
+   {
+      List<BoosterDTO> boosterList = new ArrayList<>();
+      boosters.clear();
+      UIContext uiContext = builder.getUIContext();
+      BoosterCatalogService catalogService = getCatalogService(uiContext);
+      if (catalogService == null)
+      {
+         throw new IllegalArgumentException("No BoosterCatalogService");
+      }
+      Set<Mission> missions = catalogService.getMissions();
+      for (Mission m : missions)
+      {
+         Set<Runtime> runtimes = catalogService.getRuntimes(m);
+         for (Runtime r : runtimes)
+         {
+            Optional<Booster> optional = catalogService.getBooster(m, r);
+            if (optional.isPresent())
+            {
+               Booster b = optional.get();
+               boosters.add(b);
+               boosterList.add(new BoosterDTO(b));
+            }
+         }
+      }
+      booster.setValueChoices(boosterList);
```

Comment:
`catalogService.getBoosters()` should give you a list of Boosters, without the boilerplate code above.